package.json

```
@@ -42,6 +42,7 @@
         "test:unit": "jest --verbose -c jest.config.unit.js",
         "test:integration": "./tests/run.sh",
         "flow": "flow check .",
+        "typescript": "tsc --project src/ts/types/tsconfig.json",
```

```suggestion
        "type-check": "tsc --project src/ts/types/tsconfig.json",
```
for future consistency with suite scripts

---

I'm not sure about this. Future consistency is a factor, for sure, but on the other hand I'm more concerned about the local scope here which is that we have two type systems: flow and typescript and it would be good for the script names to be explicit about that.

What about having scoped names like `type-check:flow` and `type-check:typescript`? This goes conceptually well with Suite but reflects the state of Connect.

---

ok, leave it as typescript. it's a detail which could be easily changed